server/app.js

```diff
@@ -2,8 +2,10 @@ import createError from 'http-errors';
 import express from 'express';
 import cookieParser from 'cookie-parser';
 import logger from 'morgan';
+import createDatabaseConnection from './database/connection';
 import indexRouter from './routes/index';
 
+createDatabaseConnection();
```

Comment:
Create and wait for connection only after server is started

Reply:
done